### ISSUE TITLE
WP 404 on civicrm/remoteform/ page

### ISSUE THREAD

**Issue by gah242s** [progressivetech/net.ourpowerbase.remoteform#31]
In WordPress, I'm now getting a 404 on the civicrm/remoteform/ page in the Console errors, resulting in CORS errors further down.  The CORS errors would seem to be expected as it looks like the civicrm/remoteform/ page is what grants the access, correct?  This page was definitely working on June 30th.  However, we've had WordPress 5.5 roll out since then which actually killed off the page= URLs.  I've also turned on clean URLs since then.  Based on what I saw in remoteform.php it looks like it's grabbing the URL from CiviCRM core to get to the civicrm/remoteform/ URL?  I'll admit, I'm lost on how to further diagnose this.  Any ideas?  I think Netzih is on Drupal or I'd think that recent issue and this were linked.

**Comment by netzih** [comment]
I am still on WP although playing with Drupal these days.
On my site I stopped using remote forms but on another site still using it I just updated to 5.5 and I am getting the 404 error.

**Comment by netzih** [comment]
Once thing I just noticed (at least on this site) is that when I go to the remote forms settings page I only get a submit button.
No option to add URL's.
I'll attach a screenshot. 
![de](https://user-images.githubusercontent.com/48005023/91779302-017a9580-ebaa-11ea-9faf-c6dfb44654c0.JPG)

**Comment by gah242s** [comment]
I had that happen on one site, but not the other.  I did get it to load after messing around with a few things.  (I should have made note of it.)  However, there was something in the logs that made me look somewhere else.  Did I perhaps disable the Summary Fields extension?

**Comment by netzih** [comment]
I figure out the problem I was having.
I installed an updated version of remoteform and I changed the directory name and removed "-master" from the name.
Remote form was still using the old directory name for some reason to try and pull the JS and CSS files so it was coming back 404.
Once I changed the directory name and added the -master back  to the name it took a few hours but started working again.
